config/tips.txt

```diff
@@ -132,3 +132,73 @@ Changelings can spawn as jobs that might otherwise not be spawn-able as a Traito
 Floor Tiles can be sharpened using a wirecutter to create a more deadly thrown projectile.
 Spacesuits have injection ports to administer medicine safely while in low pressure environments. These however take extra time to use.
 Setting your suit sensors to maximum isn't a good idea if you are an antagonist, as medbay can track wherever you go.
+Wrapping a bedsheet around a glass shard makes it easier to hold.
+Painkillers do not actually heal you - they just keep you standing.
+If you don't have a hazard vest, you can make armor using rail assemblies to hold together steel sheets - be aware of electrical conduction.
+A rag can hold some liquid - smothering someone's mouth will force them to breathe it in...
+You can take apart scissors with a screwdriver.
+Roll two newspapers together to swat away pests - then put a shard or a scissor blade in it...
+Syringes are delicate - do not harmfully stab someone with one, or you may transfer some bad reagents into them!
+On a small ship, cooperation is key. Kind words or a friendly PDA e-mail can make a much more fulfilling round.
```

Comment:
Honestly, I'm against these kind of 'rp' tips. They can come off quite preachy. I think we should stick to tips that are centred around gameplay mechanics